rooms:
  - name: 'salon'
    entry_pose:
      position: {x: -0.01465, y: 0.4184, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: -0.9998, w: 0.0175}
    cleaning_poses:
      - {position: {x: -0.5688, y: 4.778, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: -0.9997, w: 0.0210}}
      - {position: {x: -4.725, y: 0.225, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: 0.01096, w: 0.9999}}
      - {position: {x: -1.270, y: 1.740, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: -0.402, w: 0.915}}
    qr_data: "ROOM=LIVINGROOM"

  - name: 'mutfak'
    entry_pose:
      position: {x: 2.2317, y: 0.4571, z: 0.0}
      orientation: {x: 0.0, y: 0.0, z: 0.0175, w: 0.99984}
    cleaning_poses:
      - {position: {x: 7.026, y: 4.909, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: -0.955, w: 0.296}}
      - {position: {x: 2.776, y: 4.762, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: -0.696, w: 0.717}}
      - {position: {x: 6.848, y: 0.553, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: -0.999, w: 0.003}}
    qr_data: "ROOM=KITCHEN"

  - name: 'yatak_odasi'
    entry_pose:
      position: {x: 6.1189, y: -0.211, z: 0.0}
      orientation: {x: 0.0, y: 0.0, z: -0.6946, w: 0.7193}
    cleaning_poses:
      - {position: {x: 6.966, y: -4.683, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: 0.997, w: 0.072}}
      - {position: {x: 6.994, y: -2.400, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: 0.827, w: 0.561}}
      - {position: {x: 5.969, y: -0.439, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: 0.695, w: 0.718}}
    qr_data: "ROOM=BEDROOM"
    
  - name: 'koridor'
    entry_pose:
      position: {x: -5.0108, y: 3.3638, z: 0.0}
      orientation: {x: 0.0, y: 0.0, z: -0.9999, w: 0.005}
    cleaning_poses:
      - {position: {x: 6.90, y: 4.303, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: -0.6248, w: 0.7807}}
      - {position: {x: -6.3149, y: 1.6991, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: 0.4004, w: 0.9162}}
      - {position: {x: -5.6052, y: 3.0345, z: 0.0}, orientation: {x: 0.0, y: 0.0, z: 0.0161, w: 0.9998}}
    qr_data: "ROOM=CORRIDOR"
